openmc/data/multipole.py

```diff
@@ -112,7 +112,8 @@ def _broaden_wmp_polynomials(E, dopp, n):
         erf_beta = 1.0
         exp_m_beta2 = 0.0
     else:
-        erf_beta = np.erf(beta)
+        from scipy.special import erf
```

Comment:
Thus far we've kept scipy as an optional dependency, so scipy imports have only appeared within a function like this. I'm personally in favor of just making scipy a required dependency but I'm not sure if everyone agrees.